net.christianbeier.Gromit-MPX.yml

```diff
@@ -0,0 +1,31 @@
+id: net.christianbeier.Gromit-MPX
+runtime: org.gnome.Platform # using GNOME for the time being as we need dconf for setting hotkeys under Wayland
+runtime-version: '3.36'
```

Comment:
Update runtime to 3.38

Reply:
ACK. See 1afcbc41f20f84a3a361708a44888b080d3cb2b0